### PR TITLE
Check for duplicate root directory names

### PR DESCRIPTION
**Pull request by Kr0nox**
This PR fixes the problems caused by having two root directories with the same same as described in #1445 by throwing an exception when two last folders have the same name

### CODE REVIEW

File: core/src/main/java/de/jplag/JPlag.java [hunk 1]
```diff
@@ -96,4 +101,27 @@ private static void logSkippedSubmissions(SubmissionSet submissionSet, JPlagOpti
             }
         }
     }
+
+    private static void checkForConfigurationConsistency(JPlagOptions options) throws ConfigurationException {
+        List<String> duplicateNames = getDuplicateSubmissionFolderNames(options);
+        if (duplicateNames.size() > 0) {
+            throw new ConfigurationException(String.format("Duplicate root directory names found: %s", String.join(", ", duplicateNames)));
```
Comment (tsaglam) [inline core/src/main/java/de/jplag/JPlag.java:108]: We already have a `RootDirectoryException`, that would fit here as well.

**Review by tsaglam — CHANGES_REQUESTED** [review]
Just one small thing...

File: core/src/main/java/de/jplag/JPlag.java (hunk 1 above)
Comment (Kr0nox) [reply core/src/main/java/de/jplag/JPlag.java:108]: My plan was too keep it in line with #1479.
But if you prefer that exception I can also implement it that way

Comment (tsaglam) [reply core/src/main/java/de/jplag/JPlag.java:108]: For that PR, I want to move away from using an exception and just log and error, so thus `RootDirectoryException` fits better in your PR.

**Review by tsaglam — APPROVED** [review]

**Pull request merged**